platform_plugin_ontask/public/js/translations/en/text.js

```diff
@@ -0,0 +1,142 @@
+
```

Comment:
do we need this?

Reply:
We are not using it, I removed it: https://github.com/eduNEXT/platform-plugin-ontask/commit/27aad67434374b7e16232657482d5185c87dd7f6